WORKSPACE

```diff
@@ -32,6 +32,12 @@ nixpkgs_package(
     build_file = "@ai_formation_hazel//:BUILD.ghc",
 )
 
+nixpkgs_package(
```

Comment:
Let's use the c2hs that's built by Hazel instead of taking it from Nix.(`@haskell_c2hs//:c2hs_bin1`).  Besides that being a stress-test of Hazel, it would be less redundant since we're already building most of c2hs's dependencies in `test-packages.txt`.  I saw in the output of CircleCI:
```
these derivations will be built:
  /nix/store/1ilpd051b8xf4k1n0fcj3ngsgpzqvry1-remove-references-to.drv
  /nix/store/chj0468khz09c4jq5nvyhbb31hx61wwv-language-c-0.6.1.drv
  /nix/store/scd5mizsfak3lswpls4immhvrfwq4fwz-c2hs-0.28.2.drv
```